Bottom: e87c262ad68d196199970049bc948f612ff5b721
Top:    d1f894afded2a227c004db702db6fb291f23bbff
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 14:07:54 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index c0988cfb..6a959656 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -1,9 +1,6 @@
 ---
 # defaults file for buildbot
 
-# host containing build assets
-asset_host: 'http://{{ansible_facts.default_ipv4.address}}/'
-
 # backup host directory
 backup_hostdir: '{{ docker_backup_dir }}/buildbot'
 
@@ -78,7 +75,7 @@ image_dir: '{{ docker_projects_dir }}/buildbot-master'
 image_name: '{{ docker_registry_username }}/buildbot'
 
 # repository URI of the image source
-image_repo: https://github.com/novatechweb/docker-buildbot-master.git
+image_repo: '{{ buildbot_image_repo | default("https://github.com/novatechweb/docker-buildbot-master.git") }}'
 
 # restoration host directory
 restore_hostdir: '{{ bacula_dest }}{{ backup_hostdir }}'
